tests/tests.js

```diff
@@ -52,6 +53,66 @@ exports.defineAutoTests = function() {
                 done();
             });
         });
+
+        /*
+
+        Tests if the local copy is at the correct place and can be accessed via file plugin.
+        on android this works, as the files are copied to the PERSISTENT location:
+
+         /data/data/<app-id>/files
+
+        on iOS the files are copied to
+
+         /var/mobile/Applications/<UUID>/Library  (or .../Documents in compatibility mode)
+
+        but the persistent directory is .../Library/files.
+
+         */
+        function syncAndTest(appId, success, fail) {
+            var sync = ContentSync.sync({
+                id: appId,
+                type: 'local',
+                copyRootApp: true
+            });
+            sync.on('complete', function(localDataPath) {
+                var file = appId + '/index.html';
+                window.requestFileSystem(LocalFileSystem.PERSISTENT, 0, function(fs) {
```

Comment:
well, that is this code about. it should work the same on android and ios. once you fixed the inconsistency, the test will pass. until then, we can ignore them.